tensorflow_datasets/core/decode/base_test.py

```diff
@@ -22,7 +22,7 @@
 
 import numpy as np
 import tensorflow.compat.v2 as tf
-from tensorflow_datasets import testing
+import tensorflow_datasets as tfds
```

Comment:
Note: In `tensorflow_datasets/core/`, we do not want to import the full API, but only individual modules. This might create circular dependencies and other issues..

Reply:
Okay so should I close this pr or change some files?

Reply:
I'll apply the modifications internally. No need to do anything